docs/runbooks/deployment/prod-emergency-hotfix.md

```
@@ -0,0 +1,26 @@
+   git checkout -b hotfix/26.07.2 v26.07.1
+   ```
+4. Write and commit the fix directly on this branch.
+5. Manually push the tag to trigger deployment:
+   ```bash
+   git tag v26.07.2
+   git push origin v26.07.2
+   ```
+
+> [!WARNING]
+> **Tag format must be exactly `vYY.MM.SERIAL`** (e.g. `v26.07.2`, not `v26.7.2`). The deploy workflow validates this with a strict regex and will reject malformed tags.
+
```

markdownlint-cli2 (0.23.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)